mne/source_estimate.py

```diff
@@ -1366,6 +1367,41 @@ def plot(self, subject=None, surface='inflated', hemi='lh',
         return brain
 
     @verbose
+    def return_to_original_src(self, src_orig, subject_orig=None,
```

Comment:
@Eric89GXL @agramfort to me this is clearly remapping. We should also make sure that it's not confused with morphing as remapping just changes the vertnos while keeping the data + shape. We should call it `remap_vertno` or `remap_vertices`. I don't like `_to_src` as it implies that there are multiple choices, maybe `remap_original_vertices`.